lms/djangoapps/grades/models.py

```diff
@@ -363,3 +365,76 @@ def _prepare_params_visible_blocks_id(cls, params):
         """
         params['visible_blocks_id'] = params['visible_blocks'].hash_value
         del params['visible_blocks']
+
+
+class PersistentCourseGrade(TimeStampedModel):
+    """
+    A django model tracking persistent course grades.
+    """
+
+    class Meta(object):
+        # Indices:
+        # (course_id, user_id) for individual grades
+        # (course_id) for instructors to see all course grades
+        # (user_id) for course dashboard
```

Comment:
So the `(user_id)` index exists because of the `db_index` attribute on the `user_id` field.  And not because of this `unique_together` declaration.  Perhaps update the last line of this comment to make that clear?